src/test/packages/08-differential-package/zarf.yaml

```diff
@@ -25,4 +22,3 @@ components:
       # Do a full Git Repo Mirror
       - https://github.com/stefanprodan/podinfo.git
       - https://github.com/kelseyhightower/nocode.git
-      - "https://github.com/DoD-Platform-One/big-bang.git@refs/heads/release-1.54.x"
```

Comment:
Can we add this line back, as part of the differential test I want to make sure we are still including repo's that have referenced via branches